docs/mast/mast.rst

```diff
@@ -134,6 +134,20 @@ RA and Dec must be given in decimal degrees, and datetimes in MJD.
               image           2          GALEX ... 1000055044                0.0
               image           2          GALEX ... 1000055047 229.81061601101433
 
+We encourage the use of wildcards particularly when querying for JWST instruments
+with the instrument_name criteria. This is because of the varying instrument names
+for JWST science instruments, which you can read more about
+`here <https://outerspace.stsci.edu/display/MASTDOCS/JWST+Instrument+Names>`__.
+
+.. doctest-remote-data::
+
+   >>> from astroquery.mast import Observations
+   ...
+   >>> obs_table = Observations.query_criteria(proposal_pi="Espinoza, Nestor",
+   ...                                         instrument_name="NIRISS*")
+   >>> print(obs_table)  # doctest: +IGNORE_OUTPUT
+   {'NIRISS', 'NIRISS/IMAGE', 'NIRISS/SOSS'}
```

Comment:
Even though the output is ignored, please put in something that resembles the expected result, e.g, in this case a Table. Or, to be more useful for this case, print out the uniqued `instrument_name` column, etc.